File: scripts/migrate.py
#!/usr/bin/env python3
import subprocess
import sys
import os

# Change to project directory
os.chdir('/vercel/share/v0-project')

# Run Prisma migration
print("Running Prisma database migration...")
result = subprocess.run(['npx', 'prisma', 'migrate', 'deploy'], capture_output=True, text=True)

print(result.stdout)
if result.stderr:
    print("STDERR:", result.stderr)

if result.returncode != 0:
    print(f"Migration failed with exit code {result.returncode}")
    sys.exit(result.returncode)
else:
    print("Migration completed successfully!")

File: scripts/migrate.mjs
import { spawn } from 'child_process';

console.log('[Migration] Starting Prisma migration...');

const migration = spawn('npx', ['prisma', 'migrate', 'deploy'], {
  cwd: process.cwd(),
  stdio: 'inherit',
});

migration.on('close', (code) => {
  if (code === 0) {
    console.log('[Migration] ✓ Migration completed successfully');
    process.exit(0);
  } else {
    console.error('[Migration] ✗ Migration failed with code', code);
    process.exit(1);
  }
});
